.github/workflows/sonar.yml

```diff
@@ -32,13 +32,16 @@ jobs:
           coverage: pcov #optional, setup coverage driver
         env:
           COMPOSER_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Validate composer
+        run: composer validate --strict
+
       - name: Prepare
-        run: |
-          php -v
-          composer --version
-          composer install --dev -n --prefer-source --ignore-platform-req=php
+        run: composer install --dev -n --prefer-source --ignore-platform-req=php
```

Comment:
we can change this to `--ignore-platform-req=php+`?
this would allow higher php versions, but not lower. Available since composer 2.2.0

Reply:
sorry everybody, i tool the liberty to commit @Flyingmana's suggestion